test/dotnet.Tests/VersionTest.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+using FluentAssertions;
+
+namespace Microsoft.DotNet.Tests
+{
+    public class GivenDotnetSdk : TestBase
+    {
+        [Fact]
+        public void VersionCommandDisplaysCorrectVersion()
+        {
+            CommandResult result = new DotnetCommand()
+                    .ExecuteWithCapturedOutput("--version");
+
+            result.Should().Pass();
+            Regex.IsMatch(result.StdOut, "[0-9]{1}.[0-9]{1}.[0-9]{1}-[a-zA-Z0-9]+-[0-9]{5}").Should()
```

Comment:
Also, shouldn't there be 6 numbers at the end?

`1.0.0-rc2-002679` == `...-[a-zA-Z0-9]+-[0-9]{6}`